packages/component-submission/client/pages/SubmissionWizardHooks.js

```diff
@@ -0,0 +1,252 @@
+/**
```

Comment:
We need to pick modern react or old react and stick with it. having two in the code base is pointless

Reply:
@will-byrne / @erezmus  .... Let's talk about this